#162948798 Create routers

kwameasiago opened the pull request:
### What does this PR do?
Create two routes for  login and home page

### Description of tasks accomplished

- Installed and configured Routers
- Added two routes login and Home
- Created directories component and view respectively 
- Added dummy a component and view

### How this app should be tested?
clone this repo and checkout to ch-setup-react-router-162948798. install package requirement and run npm start to run the app

### Related pivotal tracker stories
[#162948798](https://www.pivotaltracker.com/story/show/162948798)

### Screenshots
<img width="1280" alt="screenshot 2019-02-07 at 14 27 48" src="https://user-images.githubusercontent.com/30651751/52408782-af17e680-2ae4-11e9-8823-c3c68a784656.png">

<img width="1280" alt="screenshot 2019-02-07 at 14 27 56" src="https://user-images.githubusercontent.com/30651751/52408794-b3dc9a80-2ae4-11e9-9aad-64a0100a9369.png">

kwameasiago committed:
Chore(Router) Create router
- Install router version 4.3.1
- Add two routes Home and Login
- Create dummy component and view
[Deliver #162948798]

Georgeygigz commented on src/components/Login.jsx:
```diff
@@ -0,0 +1,12 @@
+        )
+    }
+}
+export default Login;
```
Add a blank line at the end this file

Georgeygigz commented on src/view/Home.jsx:
```diff
@@ -0,0 +1,12 @@
+        )
+    }
+}
+export default Home;
```
ditto

MaggieChege reviewed (approved):
Looks good to me

Georgeygigz reviewed (approved):
Looks good to me

KMaina commented on src/App.jsx:
```diff
@@ -1,30 +1,21 @@
-        </header>
-      </div>
+      <BrowserRouter>
+      {/* <div> */}
```
Kindly remove this line of code that is commented

KMaina commented on src/App.jsx:
```diff
@@ -1,30 +1,21 @@
+          <Route path='/login' component={Login}></Route>
+        </Switch>
+       
+      {/* </div> */}
```
ditto

cosmas28 commented on src/App.jsx:
```diff
@@ -1,30 +1,18 @@
-          </a>
-        </header>
-      </div>
+      <BrowserRouter>
```
It's a good practice to create a route component on it's on file. ie `src/routes/routes.js`. Therefore, move these dummy routes to their own directory and nest the `Route component` as a child component of the `App component`.

kwameasiago replied:
Ok will do that

kwameasiago committed:
Chore(Routes) Relocate Component
- Create a new file Router.jsx
- Move routes component to Route.jsx
[Delivers #162948798]

cosmas28 commented on src/components/routes.jsx:
```diff
@@ -0,0 +1,19 @@
+import Home from '../view/Home';
+import Login from './Login';
+
+class Routes extends Component {
```
Any reason why this is a class-based component?

kwameasiago replied:
No reason. Just preference, Should I change to functional components?

cosmas28 reviewed (changes requested):
I requested the `Route component` to be under `src/routes/routes.jsx` tree. Why did you move it to `src/components`?

kwameasiago committed:
chore (Routes) Use function components
- Change stateless class based components to functional components
[Delivers #162948798]

cosmas28 commented on src/view/Home.jsx:
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Home = () => {
+        return (
```
Kindly fix the indentation from line 4 to line 8.

cosmas28 commented on src/view/Home.jsx:
```diff
@@ -0,0 +1,10 @@
+            </div>
+        );
+};
+export default Home;
```
There should be a line space before this line. This makes your code more readable.

cosmas28 replied:
The indentation space should be consistent in all components. Kindly use two spaces like the way it is on `Login` and `Routes` components.

kwameasiago committed:
feat(Routes) Fix linting errors
- Add a line at line 10
- Fix Indentations error
[Delivers #162948798]
